rust_src/src/frames.rs

```diff
@@ -507,13 +521,13 @@ pub fn next_frame(frame: LispFrameOrSelected, miniframe: LispObject) -> LispFram
     // a valid candidate will be returned regardless of its position.
     while passed < 2 {
         for_each_frame!(f => {
-	    if passed > 0 {
-	        let tmp = unsafe { candidate_frame(f.into(), frame_obj, miniframe) };
-	        if !tmp.is_nil() {
+        if passed > 0 {
```

Comment:
Are you using the correct `rustfmt` here? Why would it drop the indentation??

Reply:
The change stems from a tab literal being changed to four spaces, so the visible difference is a quirk of tabs being displayed as 8 spaces wide. I don't know how the tab literals got in there in the first place.

Reply:
That was likely me hacking in vi while I had a busted remacs binary :-)

Reply:
My merge of #1240 should resolve this for everyone.

Reply:
You should rebase. This issue is merged.